webapp/packages/core-authentication/src/UsersResource.ts

```diff
@@ -240,6 +242,7 @@ export class UsersResource extends CachedMapResource<string, AdminUser, UserReso
     }
 
     const usersList: AdminUser[] = [];
+    const pages: Parameters<typeof this.offsetPagination.setPage>[] = [];
```

Comment:
im not sure that we need it, because we don't and can't use it anywhere to declare exact implementation, instead we have several places when we need to "delay" `setPage` call (we want to call it synchronously in same action with data mutation and after data mutation)

Probably we can want in future ability to delay setPage call via API but not with this array of arguments + for each + call